src/bun.js/bindings/JSSink.cpp

```diff
@@ -1,6 +1,6 @@
 
 // AUTO-GENERATED FILE. DO NOT EDIT.
-// Generated by 'make generate-sink' at 2023-06-14T21:38:04.394Z
```

Comment:
did you mean to update webkit (above)?

Reply:
I just run `make codegen` to update the generated classes